install/helm/open-match/values.yaml

```diff
@@ -118,14 +118,22 @@ redis:
     enabled: true
   cluster:
     slaveCount: 2
+  serviceAccount:
+    create: true
+    name: open-match-redis-service
   # Redis may require some changes in the kernel of the host machine to work as expected, 
   # in particular increasing the somaxconn value and disabling transparent huge pages.
-  # TODO: disable THP - requires priviledged service account, blocked by current PSP setup.
   # https://github.com/helm/charts/tree/master/stable/redis#host-kernel-settings
-  securityContext:
-    sysctls:
-    - name: net.core.somaxconn
-      value: "10000"
+  sysctlImage:
+    enabled: true
+    mountHostSys: true
+    command:
+      - /bin/sh
```

Comment:
Add a link to https://github.com/helm/charts/tree/master/stable/redis#host-kernel-settings.